bin/phpspec

```diff
@@ -1,26 +1,28 @@
 #!/usr/bin/env php
 <?php
 
-define('PHPSPEC_VERSION', '3.0.x@dev');
```

Comment:
people might be relying on the constant to detect the PhpSpec version in their suite. Is it intended that it is not exposed to the code anymore ?